webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -1889,6 +1892,71 @@ var userOverride = {
                     window.Applozic.ALSocket.events.onUserDeactivated =
                         events.onUserDeactivated;
                 }
+                if (typeof events.onChatWidgetOpen === 'function') {
```

Comment:
instead of writing each and every event below, can something be done so that all events things become part of kmWidgetEvents in just few lines?